hack/milvus_cluster_values.yaml

```diff
@@ -0,0 +1,28 @@
+cluster:
+  enabled: true
+
+etcd:
+  enabled: true
+
+pulsar:
+  enabled: true
+
+minio:
+  enabled: false
+
+externalS3:
+  enabled: true
+  # TODO(guangrui): revisit. It seems only buckets in us-east-1 works.
```

Comment:
Oh i see. Hm, weird.

Reply:
yah, weird. Somehow `querycoord` always looks for buckets in `us-east-1`, while other components can take bucket in other regions.